monai/bundle/scripts.py

```diff
@@ -489,6 +490,12 @@ def patch_bundle_tracking(parser: ConfigParser, settings: dict):
                     handlers.append(v)
         elif k not in parser:
             parser[k] = v
+    # save the executed config into file
+    timestamp = time.strftime('%Y%m%d_%H%M%S')
+    filename = parser["output_dir"] + f"/config_{timestamp}.json"
+    # experiment management tools can refer to this config item to track the config info
+    parser["execute_config"] = filename
+    parser.export_config_file(parser.get(), filename)
```

Comment:
Hi @Nic-Ma ,
     A `path doesn't exist` error would be reported, if the `eval` folder doesn't exists. And the original spleen bundle downloaded from model zoo doesn't have this folder.

Thanks,
Bin

Reply:
Updated in the latest commit, could you please help try it again?

Thanks.